tests/Cucumber/CompatibilityTest.php

```diff
@@ -54,13 +54,6 @@ class CompatibilityTest extends TestCase
         'incomplete_background_2.feature' => 'Background descriptions not supported',
     ];
 
-    /**
-     * @var array<string, string>
-     */
-    private array $parsedButShouldNotBe = [
-        'invalid_language.feature' => 'Invalid language is silently ignored',
-    ];
-
```

Comment:
I think we'd keep this, but based on #349, and just remove the invalid_language.feature from the `gherkin-32` set.

Reply:
I re-added this for now. It will handled in a follow-up PR (once both the dialect API and the compatibility mode are merged).